apps/web-giddh/src/app/purchase/purchase-order-preview/purchase-order-preview.component.scss

```diff
@@ -0,0 +1,929 @@
+    background-image: url("../../../assets/images/searchIcon.svg");
+    background-position: 10px 13px;
+    background-repeat: no-repeat;
+}
+
+.invoice-search-box button {
+    margin-left: 10px;
+}
+
+.invoice-search-box {
+    display: flex;
+    padding: 15px 15px 10px;
+    justify-content: space-around;
+}
+
+.invoice-sidebar-wrapper {
+    box-shadow: 3px 3px 6px rgba(0, 0, 0, 0.1);
+    z-index: 9;
+}
+
+.invoice-detail-wrapper {
+    height: 100vh;
+}
+
+.invoice-editWrapper {
+    height: 100vh;
+    overflow-y: auto;
+    .invoice-editHeader {
+        display: flex;
+        justify-content: space-between;
+        height: 54px;
+        background-color: $white;
+        border-top: 1px solid $gainsboro;
+        border-bottom: 1px solid $gainsboro;
+        align-items: center;
+        span {
+            padding: 10px;
+            p {
+                font-size: 16px;
+                font-weight: bold;
+            }
+        }
+        span.close-modal {
+            height: 32px;
+            width: 32px;
+            background-color: $very-light-gray;
+            border: 1px solid $gainsboro;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            margin-right: 10px;
+        }
+    }
+    .invoice-editBody {
+        padding-bottom: 140px;
+    }
+}
+
+.change-status-dropdown #setting-tab {
+    width: 340px;
+    left: auto;
+    right: 0;
+}
+
+.sendEmailModal {
+    min-width: 240px;
+    position: absolute;
+    top: 40px;
+    z-index: 10;
+    right: 0;
+    border: 1px solid $gainsboro;
+    .sendEmailModalWrapper {
+        background-color: $white;
+        box-shadow: 0 0 6px rgba(0, 0, 0, 0.2);
+        height: 200px;
+        .sendEmailModalHeader {
+            background-color: $whisper-color;
+            padding: 8px 10px;
+            border-bottom: 1px solid $gainsboro;
+            span {
+                width: unset;
+            }
+        }
+    }
+    .sendEmailModalBody {
+        padding: 5px;
+    }
+    .sendEmailModalBody label {
+        font-size: 12px;
+        margin-bottom: 7px;
+    }
+    .btn-success {
+        background: $success-green;
+        color: $white;
+    }
+    .aqua-color a {
+        color: $deep-sky-blue;
+        font-size: 12px;
+    }
+}
+
+.dropdown-width-small {
+    width: 165px;
+}
+
+.dropdown-btn-new {
+    padding: 5px 16px !important;
+}
+
+.upload-excel {
+    display: flex;
+    height: 320px;
+    position: relative;
+    align-items: center;
+    justify-content: center;
+}
+
+.upload-excel {
+    max-width: 500px;
+}
+
+.dashed-border {
+    border: 2px dashed $gainsboro;
+}
+
+#csv-upload-input {
+    opacity: 0;
+    position: absolute;
+    left: 0;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    height: 100%;
+    z-index: 2;
+}
+
+.select-file {
+    color: $success-green;
+}
+
+.file-uplod-penal {
+    width: 100%;
+    text-align: left;
+    border: 1px solid gainsboro;
```

Comment:
Incorrect format of accessing variable use `$gainsboro`

Reply:
done